src/lang/pt_BR.json

```diff
@@ -0,0 +1,751 @@
+  "Dashboard.header": "Dashboard",
+  "Home.FeaturedChallenges.improveOSM": "Comece a melhorar o OpenStreetMap!",
+  "Home.FeaturedChallenges.start": "Start",
+  "Leaderboard.title": "Leaderboard",
+  "Leaderboard.global": "Global",
+  "Leaderboard.scoringMethod.label": "Método de pontuação",
+  "Leaderboard.scoringMethod.explanation": "##### Pontos são concedidos por tarefa concluída da seguinte forma:\n\n| Status        | Points |\n| :------------ | -----: |\n| Fixed         | 5      |\n| Not an Issue  | 3      |\n| Already Fixed | 3      |\n| Too Hard      | 1      |\n| Skipped       | 0      |",
+  "Leaderboard.user.points": "Pontos",
+  "Leaderboard.user.topChallenges": "Principais Desafios",
+  "Leaderboard.users.none": "Nenhum usuário por período de tempo",
+  "Leaderboard.controls.loadMore.label": "Mostrar mais",
+  "Leaderboard.updatedFrequently": "Atualizado a cada 15 minutos",
+  "Leaderboard.updatedDaily": "Atualizado a cada 24 horas",
+  "Profile.settings.header": "Configurações do usuário",
+  "Profile.userSince": "Usuário desde:",
+  "Profile.form.defaultEditor.label": "Editor padrão",
+  "Profile.form.defaultEditor.description": "Selecione o editor padrão que você deseja usar ao corrigir tarefas. Ao selecionar esta opção, você poderá pular a caixa de diálogo de seleção do editor depois de clicar em editar em uma tarefa.",
+  "Profile.form.defaultBasemap.label": "Mapa Base Padrão",
+  "Profile.form.defaultBasemap.description": "Selecione o mapa base padrão para exibir no mapa. Apenas um mapa base de desafio padrão substituirá a opção selecionada aqui.",
+  "Profile.form.customBasemap.label": "Basemap personalizado",
+  "Profile.form.customBasemap.description": "Insert a custom base map here. E.g. `https://\\{s\\}.tile.openstreetmap.org/\\{z\\}/\\{x\\}/\\{y\\}.png` {dummy}",
+  "Profile.form.locale.label": "Locale",
+  "Profile.form.locale.description": "Localidade do usuário para usar na interface do usuário do MapRoulette.",
+  "Profile.form.leaderboardOptOut.label": "Desativar o placar" ,
+  "Profile.form.leaderboardOptOut.description": "Se sim, você ** não ** aparecerá na tabela de classificação pública.",
+  "Profile.apiKey.header": "Chave de API",
+  "Profile.apiKey.controls.copy.label": "Copiar",
+  "Profile.apiKey.controls.reset.label": "Reset",
+  "Activity.action.updated": "Atualizado",
+  "Activity.action.created": "Criado",
+  "Activity.action.deleted": "Excluído",
+  "Activity.action.taskViewed": "Visto",
+  "Activity.action.taskStatusSet": "Definir status no",
+  "Activity.action.tagAdded": "Tag adicionado para",
+  "Activity.action.tagRemoved": "Tag removido de",
+  "Activity.action.questionAnswered": "Pergunta respondida em",
+  "Activity.item.project": "Projeto",
+  "Activity.item.challenge": "Desafio",
+  "Activity.item.task": "Tarefa",
+  "Activity.item.tag": "Tag",
+  "Activity.item.survey": "Inquérito",
```

Comment:
Pesquisa

Reply:
Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!